experiment/measurer.py

```diff
@@ -67,13 +68,131 @@ def exists_in_experiment_filestore(path: pathlib.Path) -> bool:
                               must_exist=False).retcode == 0
 
 
+def get_fuzzer_benchmark_key(fuzzer: str, benchmark: str):
```

Comment:
At some point i'd like it if the coverage intersection stuff were done in another module. The main reason is because it is cleaner. But also I'm making huge changes to measurer in #570 so putting more things here will make that harder.

Reply:
Do you mean regions covered by multiple fuzzers when you say coverage intersection? This CI only extracts the location information of covered regions and stores them in a file for later calculation. The actual calculation of intersection or uniquely covered regions is placed in the front-end when rendering the html.

Reply:
Yes i have talked to @Shadoom7 , he will start moving things into a new module like common/coverage_utils.py or something in next CL.